osbs/build/build_requestv2.py

```diff
@@ -402,10 +447,14 @@ def set_data_from_reactor_config(self, reactor_config_data):
             if self.user_params.flatpak.value and not self.user_params.base_image.value:
                 raise OsbsValidationException(
```

Comment:
Should we always raise `OsbsValidationException` if `reactor_config_data` is falsey?

Reply:
well it will fail anyway few lines bellow if it doesn't fine source_registry